### DISCUSSION TITLE
bale wrapper

### DISCUSSION THREAD

**Discussion by BRADP1980** [Courseplay/Courseplay_FS22#2494] [Q&A] [answered]
I started my bale wrapper on a field with courseplay and it finished but when i went to a new field to wrap the bales on that field it tells me CP: vehicle is to far away from field. any idea? i am on the newest version. TIA

**Comment by Tensuko** [comment, marked as answer]
It is written in the changelog!
`Bale collector/wrapper now works the same way, as the combine unloader in terms of markers.

    The field position is now saved into the vehicle and will not be reset to the vehicle position, when started by the hud!
        The position has to be set everytime in the ai menu!
        This fixes problems with AD restarting and also has the advantage that you don't need another AD Target on the field to start the driver.
`
